tests/performance/load-tests/samples/simple-with-pvc.yaml

```diff
@@ -0,0 +1,13 @@
+kind: DevWorkspace
```

Comment:
the name is misleading should  probably be `simple-with-editor`

Reply:
I have added new samples. Review it, please.